dune_api_scripts/store/solver_list.py

```diff
@@ -0,0 +1,47 @@
+"""Modifies and executed dune query for today's data"""
+from __future__ import annotations
+
+import json
+import os
+
+from duneapi.api import DuneAPI
+from duneapi.types import DuneQuery, Network
+
+
+SOLVER_QUERY = """
+select 
+    concat('0x', encode(address, 'hex')) as address,
+    environment,
+    name, 
+    active
+from gnosis_protocol_v2.view_solvers
+"""
+
+SHORT_NAMES = {
```

Comment:
You could create a constant.py file and start putting common constants in there

Reply:
Yes this is something we can do for sure,but at this particular moment, these constants are not needed outside the scope of this file... I will do it anyway.

Reply:
Done.